mne/forward/tests/test_field_interpolation.py

```diff
@@ -158,6 +159,16 @@ def test_make_field_map_meg():
 
     assert_raises(ValueError, make_field_map, evoked, ch_type='foobar')
 
+    # now test the make_field_map on head surf for MEG
+    evoked.pick_types(meg=True, eeg=False)
+    fmd = make_field_map(evoked, trans_fname, meg_surf='head',
+                         subject='sample', subjects_dir=subjects_dir)
+    assert_true(len(fmd) == 1)
+    assert_array_equal(fmd[0]['data'].shape, (642, 106))  # maps data onto surf
+    assert_true(len(fmd[0]['ch_names']), 106)
+
+    assert_raises(ValueError, make_field_map, evoked, meg_surf='foobar')
```

Comment:
This still needs `subjects_dir=subjects_dir`, otherwise *that* creates an error before you hit your `ValueError` check